### DISCUSSION TITLE
找不到setupenv.cmd文件

### DISCUSSION THREAD

**Discussion by yuyijie1995** [apache/incubator-mxnet#13987] [Q&A]
![image](https://user-images.githubusercontent.com/30818917/51727046-22f9ce00-20a5-11e9-8fa9-74fa3cf7d804.png)
下载页面只能下载源码，并没有预编译包，请问是怎么回事

[a comment by an automated account is not shown]

**Comment by frankfliu** [comment]
@mxnet-label-bot add [build, question]

**Comment by frankfliu** [comment]
@yuyijie1995 
If you want to install mxnet binary for python, you should use following command: 
```
pip install mxnet
```

See: http://mxnet.incubator.apache.org/versions/master/install/index.html?platform=Linux&language=Python&processor=CPU
